Code/Tools/TestImpactFramework/Runtime/Python/Code/Source/Dependency/TestImpactPythonTestSelectorAndPrioritizer.h

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <Dependency/TestImpactTestSelectorAndPrioritizer.h>
+#include <Target/Python/TestImpactPythonProductionTarget.h>
+#include <Target/Python/TestImpactPythonTestTarget.h>
+
+namespace TestImpact
+{
+    class PythonTestSelectorAndPrioritizer
```

Comment:
update: An API comment here would likely be useful

Reply:
Fixed 👍